buildstockbatch/gcp/gcp.py

```diff
@@ -634,6 +634,32 @@ def run_batch(self):
         logger.info(f"Batch job status: {job_status}")
         if job_status != "SUCCEEDED":
             raise RuntimeError("Batch job failed. See GCP logs at {job_url}")
+        else:
+            task_group = job_info.task_groups[0]
+            task_spec = task_group.task_spec
+            instance = job_info.status.task_groups["group0"].instances[0]
+
+            # Format stats for easy copying into a spreadsheet
+            keys, values = [], []
+
+            def append_stat(key, value):
+                nonlocal keys
+                nonlocal values
+                width = max(len(str(key)), len(str(value)))
+                keys.append(str(key).rjust(width))
+                values.append(str(value).rjust(width))
+
+            append_stat("Simulations", n_sims)
+            append_stat("Tasks", task_group.task_count)
+            append_stat("Parallelism", task_group.parallelism)
+            append_stat("milliCPU/task", task_spec.compute_resource.cpu_milli)
+            append_stat("MiB/task", task_spec.compute_resource.memory_mib)
+            append_stat("Machine type", instance.machine_type)
+            append_stat("Provisioning", instance.provisioning_model.name)
```

Comment:
Many of these values just come from config, and more values makes logged lines longer and harder to read, especially if they're wrapped. How about logging the parameters separately from the calculated values (e.g., tasks) and stats?

Reply:
My goal was to make it really easy to copy all this info to this spreadsheet, ideally with a single copy/paste: https://docs.google.com/spreadsheets/d/1CoNwzVT_9r6KIXinwd0_PoCtfgL8y470AKsyf7UtJKE/edit#gid=290943411

Reply:
Got it. Would it make sense to log this as multi-line output then so it's easier to copy without the log entry heading?

Reply:
Ah, got it!